app/code/Apruve/Payment/Controller/updateOrderStatus/Index.php

```diff
@@ -50,26 +50,25 @@ public function execute()
         $success = false;
 
         switch ($action) {
-            case 'invoice.closed':
-                $success = $this->_invoiceClosed($data);
+            case 'invoice.funded':
```

Comment:
We need to handle both `invoice.funded` and `invoice.closed`.  If an invoice is self-financed, we will never send an `invoice.funded` webhook for it because there will be no creditor to fund it.  The invoice should get closed out whenever either webhook is received.  If we get either webhook after the invoice is already closed out, we should ignore it.